modules/ztp-setting-managed-bare-metal-host-kernel-arguments-gitops-workflow.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc
+
+[id="setting-managed-bare-metal-host-kernel-arguments_{context}"]
+= Setting managed bare-metal host kernel arguments
+
+Specify kernel arguments for the Discovery ISO to customize the boot process in managed bare-metal hosts. This is useful for clusters with specific environmental requirements. For example, configure the `rd.net.timeout.carrier` kernel argument to facilitate static networking or to receive a DHCP address before downloading the root file system. 
+
+[NOTE]
+====
+In {product-title} {product-version}, you can only add kernel arguments. You can not replace or delete kernel arguments.
+====
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (oc).
+* You have logged in to the hub cluster as a user with cluster-admin privileges.
+
+.Procedure
+
+. Create the `InfraEnv` CR and edit the `spec.kernelArguments` specification to configure kernel arguments.
+
+.. Save the following YAML in an `InfraEnv-example.yaml` file:
++
+[NOTE]
+====
+The `InfraEnv` CR has template variables such as `{{ .Cluster.ClusterName }}`. The `SiteConfig` CR automatically populates values for these templates during deployment. Do not edit the templates manually.  
```

Comment:
It's not that the `InfraEnv` has template variables, it's that you can use them when applying the CR to individual hosts. The templates pull details from, for example a `ConfigMap` CR that stores IP addresses, NIC details, etc. See https://55012--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.html#ztp-using-hub-cluster-templates_ztp-advanced-policy-config

So a single `InfraEnv` CR can be applied to a fleet of clusters using templates. Saves having to create CRs for every host.

Reply:
See Ian's [comment](https://github.com/openshift/openshift-docs/pull/55012#discussion_r1090780129). I don't know where the templates are defined, only that they are substituted with these values when SiteConfig kicks off.